service/src/test/java/com/ge/predix/acs/privilege/management/dao/GraphResourceRepositoryTest.java

```diff
@@ -17,19 +31,20 @@
 import org.apache.tinkerpop.gremlin.structure.Graph;
 import org.apache.tinkerpop.gremlin.structure.Vertex;
 import org.apache.tinkerpop.gremlin.util.iterator.IteratorUtils;
+import org.testng.Assert;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 import com.ge.predix.acs.config.GraphConfig;
+import com.ge.predix.acs.rest.Parent;
+import com.ge.predix.acs.utils.JsonUtils;
 import com.ge.predix.acs.zone.management.dao.ZoneEntity;
 import com.thinkaurelius.titan.core.SchemaViolationException;
 import com.thinkaurelius.titan.core.TitanFactory;
 
 public class GraphResourceRepositoryTest {
-
```

Comment:
Delete tests should lookup the vertex after delete and assert that is not found.  (in addition to existing asserts on count)